lib/JMAP/Tester/Role/Result.pm

```diff
@@ -23,6 +23,20 @@ has http_response => (
   is => 'ro',
 );
 
+=method response_payload
+
+Returns the raw payload of the response, if there is one. Empty string
```

Comment:
The English here isn't great, if you care. ;-)

Reply:
Got an alternate suggestion? (I'm honestly not sure what you don't like about it :))

Reply:
12d later, I have no idea.